recipe/meta.yaml

```diff
@@ -19,22 +19,25 @@ build:
 
 requirements:
   build:
-    # python is a build requirement on Windows to resolve features.
-    - python           # [win]
+    # for feature activation
+    - vc  9  # [win and py27]
+    - vc 10  # [win and py34]
+    - vc 14  # [win and py35]
```

Comment:
I believe re-rendering uses the Python dependency to create the build matrix. And you be unable to do so in that case. You can test by trying to re-render this PR.